### ISSUE TITLE
[Release process] Doc tests are neglected

### ISSUE THREAD

**Issue by ArturNiederfahrenhorst** [ray-project/ray#36521]
### Description

Our Doc tests _(":book: Documentation")_ become red quite often.
There is often not enough sense of ownership for the teams who own the failing tests.
This is likely caused by a combination of the following:

1. The test is often red, so not a reliable signal that your PR causes an issue
2. The logs are very large and hard to parse
3. The test is shared between all teams

Just as a reminder which jobtest is referenced here, it's this one:
<img width="237" alt="Screenshot 2023-06-16 at 15 32 50" src="https://github.com/ray-project/ray/assets/9356806/8d0ea3a9-f3c7-47c0-92cd-976d0f3d74db">

**Comment by hora-anyscale** [comment]
@maxpumperla @angelinalg - thoughts on how to improve the process for doc release tests?

**Comment by dstrodtman** [comment]
Closing as timed out.
